tests/lexer_test.py

```diff
@@ -59,8 +59,8 @@ def test_function_call_template(self) -> None:
         # have so we can assert it is what we expected.
         self.assertEqual(mock.grammar, expected_dict)
 
-    def test_random(self) -> None:
-        '''This is just the first thing I tried so it stayed here'''
+    def test_grammar_parsing(self) -> None:
```

Comment:
I agree my name was bad but I'm not sure this is much better because all tests in this class will test grammar parsing :/

Reply:
Done, see commits below.